rtv/content.py

```diff
@@ -122,7 +122,7 @@ def strip_praw_submission(sub):
         data['score'] = '{} pts'.format(sub.score)
         data['author'] = name
         data['permalink'] = sub.permalink
-        data['subreddit'] = strip_subreddit_url(sub.permalink)
+        data['subreddit'] = str(sub.subreddit)
```

Comment:
Awesome, thanks for checking. I agree that your way is a lot nicer. :+1: praw is currently a *very* active project and sometimes it's hard to keep up with all of the changes going on over there.